code/day2-operations/deployment/k8s/prometheus_values.yaml

```diff
@@ -9,42 +9,95 @@ serviceAccounts:
 alertmanager:
   enabled: false  ## If false, alertmanager will not be installed
 
-nodeExporter:     
+prometheus-node-exporter:
   enabled: false  ## If false, node-exporter will not be installed
 
-pushgateway:
+prometheus-pushgateway:
   enabled: false  ## If false, pushgateway will not be installed
 
-kubeStateMetrics:
+kube-state-metrics:
   enabled: false  ## If false, kube-state-metrics sub-chart will not be installed
 
 server:
   global:
     scrape_interval: 15s  ## How frequently to scrape targets by default
 
+  #extraArgs:
+  ## Additional Prometheus server container arguments
+  ##
+    #config: /etc/otel/config.yaml  
+
+  ## Additional Prometheus server Volume mounts
+  ##
+  extraVolumeMounts:
+    - mountPath: /etc/istio-output-certs
+      name: istio-certs
+      readOnly: true
+
+  ## Additional Prometheus server Volumes
+  ##
+  extraVolumes:
+    - name: istio-certs
+      emptyDir: {}  
+
   persistentVolume:
     ## If true, Prometheus server will create/use a Persistent Volume Claim
     ## If false, use emptyDir
     enabled: false    
-
+  podAnnotations:
+    ## Annotations to be added to Prometheus server pods
+    ##  
+    proxy.istio.io/config: |
+      # configure an env variable `OUTPUT_CERTS` to write certificates to the given folder
+      proxyMetadata:
+        OUTPUT_CERTS: /etc/istio-output-certs
+    sidecar.istio.io/userVolumeMount: '[{"name": "istio-certs", "mountPath": "/etc/istio-output-certs"}]'
+    traffic.sidecar.istio.io/includeInboundPorts: ""
+    traffic.sidecar.istio.io/includeOutboundIPRanges: ""
+  podLabels:
+    ## Labels to be added to Prometheus server pods
+    ##
+    sidecar.istio.io/inject: "true"        
```

Comment:
needs for prometheus to communicate with day2-service through istio service mesh